app/models/post.rb

```diff
@@ -3,6 +3,9 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, comparison: { greater_than: -1 }
+  validates :likes_counter, comparison: { greater_than: -1 }
```

Comment:
- [ ] [OPTIONAL] You can use the greater_than_or_equale_to: 0, instead of greater_than: -1. this is just a personal pereferance so this suggestion is not required